### ISSUE TITLE
rpm_set_filecaps.sh expects a different format from its rpm command

### ISSUE THREAD

**Issue by ebryerwork** [xcat2/xcat-core#7290]
In the rpm_set_filecaps.sh script is an `rpm` command piped into `gawk`
```
rpm --qf "[%{FILECAPS} %{FILENAMES}\n]" -qa \
   | gawk -v "verify=$verify" -v "quiet=$quiet" -v "errflag=.ERROR." '
$1 == "=" {
   caps = $2
```
The output format of that `rpm` command changed between RHEL 8.6 and RHEL 8.7. Whereas in <=8.6 that command would output
```
= cap_setgid+ep /usr/bin/newgidmap
= cap_setuid+ep /usr/bin/newuidmap
```
on 8.7 it outputs
```
cap_setgid=ep /usr/bin/newgidmap
cap_setuid=ep /usr/bin/newuidmap
```
As a result, the capabilities on those files are not being set later in the script.

$ lsxcatd -a
Version 2.16.1 (git commit a85566c8be85ee7eec991b0840e53295de686e49, built Wed Nov  4 16:55:06 EST 2020)

Unrelated, there is a typo on line 19 of that script: `quite=""` instead of `quiet=""`.

**Comment by ebryerwork** [comment]
I got the idea to check for that file in the repo master. That file isn't part of the xcat repo, so I'm closing this.

**Comment by gurevichmark** [comment]
@ebryerwork Thanks for pointing out the possibility of `quite` typo. I did find an instance of it in `xcat-core`. Fixed by #7292
